RELEASE/scripts/autoscend.ash

```diff
@@ -1,4 +1,4 @@
-since r27678;	// fix: identify burning leaves in small
+since r27722;	// fix: identify burning leaves in small
```

Comment:
```suggestion
since r27722;	// feat: preferences candy cane options
```

Not strictly necessary but keeping comments up to date helps the next person.